### PR TITLE
Responsive window changes, and titles in all three graphs

### PR DESCRIPTION
**Pull request by xavierdelacruz**
- Made it responsive based on browser size
- Made titles very explicit

### CODE REVIEW

File: visualization/commit-versus-coverage.html [hunk 1]
```diff
@@ -58,250 +85,269 @@
...
+                    .attr("transform", "rotate(90)")
+                    .style("text-anchor", "start");
+
+                svg.append("g")
+                    .append("text")
+                    .attr("x", width / 2)
+                    .attr("y", -10)
+                    .attr("transform", "rotate(0)")
+                    .style('font', "sans-serif")
+                    .style("font", "14px times")
```
Comment (jregchiu) [inline visualization/commit-versus-coverage.html]: Do we need two different font options to be set?

File: visualization/commit-versus-testrunmemory.html [hunk 2]
```diff
@@ -31,276 +31,309 @@
...
+                    .attr("transform", "rotate(90)")
+                    .style("text-anchor", "start");
+
+                svg.append("g")
+                    .append("text")
+                    .attr("x", width / 2)
+                    .attr("y", -10)
+                    .attr("transform", "rotate(0)")
+                    .style('font', "sans-serif")
+                    .style("font", "14px times")
```
Comment (jregchiu) [inline visualization/commit-versus-testrunmemory.html]: Do the two different font options conflict?

File: visualization/commit-versus-teststatus.html [hunk 3]
```diff
@@ -12,174 +12,223 @@
...
+                    .style("text-anchor", "end")
+                    .text("Number of Tests");
+
+                svg.append("g")
+                    .append("text")
+                    .attr("x", width / 2)
+                    .attr("y", 0)
+                    .attr("transform", "rotate(0)")
+                    .style('font', "sans-serif")
+                    .style("font", "14px times")
```
Comment (jregchiu) [inline visualization/commit-versus-teststatus.html]: I noticed that the axis labels used sans-serif so guessing that's what you intended?

**Review by jregchiu — APPROVED** [review]

**Comment by xavierdelacruz** [comment]
Merging Thanks @jregchiu !

**Pull request merged**